### DISCUSSION TITLE
Load library into .net framework

### DISCUSSION THREAD

**Discussion by ExpertInvest** [doghappy/socket.io-client-csharp#211] [Q&A] [answered]
Hi, I can't load that project into my .net framework project.
it says: 

> "Could not install package 'SocketIOClient 3.0.1'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.5', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author."		

Any idea if is it possible?
Thanks

**Comment by doghappy** [comment, marked as answer]
The library is based on .net standard 2.0. This means you can't use it on .net framework 4.5.

You can upgrade your project from VS, just modify the Target Framework to .net framewrok 4.6.1 

https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support

**Reply by ExpertInvest** [reply]
Thanks, 
it's working.
Very good library!
